test/plots/waterfallPlotTests.ts

```diff
@@ -1,123 +1,251 @@
+        yScale = new Plottable.Scales.Linear();
+        plot = new Plottable.Plots.Waterfall<string, number>();
+        plot.x((d) => d.x, xScale);
+        plot.y((d) => d.y, yScale);
+        plot.total((d) => d.t);
+        plot.addDataset(dataset);
+      });
+
+      it("does not have connectors enabled by default", () => {
+        assert.isFalse(plot.connectorsEnabled(), "no connectors by default");
+        plot.destroy();
+        svg.remove();
+      });
+
+      it("can set if connectors are enabled", () => {
+        assert.strictEqual(plot.connectorsEnabled(true), plot, "setter returns calling object");
+        plot.renderTo(svg);
+        let bars = plot.content().selectAll("rect");
+        let connectors = plot.content().selectAll("line.connector");
+        assert.strictEqual(bars.size(), connectors.size() + 1, "there is one more bar than number of connectors");
+        connectors.each(function(datum, index) {
+          let connector = d3.select(this);
+          let bar = d3.select(bars[0][index]);
+          let connectorOnBottom = bar.classed("waterfall-decline");
+          if (connectorOnBottom) {
+            assert.closeTo(numAttr(connector, "y1"), numAttr(bar, "y") + numAttr(bar, "height"),
+              window.Pixel_CloseTo_Requirement, "connector on declining bar at bottom");
           } else {
-            assert.isTrue(selection.classed("waterfall-decline"));
+            assert.closeTo(numAttr(connector, "y1"), numAttr(bar, "y"),
+              window.Pixel_CloseTo_Requirement, "connector on non-declining bar at top");
           }
-        }
+          assert.strictEqual(numAttr(connector, "y1"), numAttr(connector, "y2"), "connector stays at same height");
+        });
+        plot.destroy();
+        svg.remove();
       });
-      svg.remove();
-    });
 
-    it("renders connector lines correctly", () => {
-      plot.connectorsEnabled(true);
-      plot.renderTo(svg);
-      let bars = renderArea.selectAll("rect")[0];
-      let connectors = renderArea.selectAll("line.connector")[0];
-      assert.isTrue(bars.length - 1 === connectors.length, "there is one more bar than number of connectors");
-      connectors.forEach((connector, index) => {
-        let selection = d3.select(connector);
-        let firstBar = d3.select(bars[index]);
-        let secondBar = d3.select(bars[index + 1]);
-        let firstY = +firstBar.attr("y");
-        let firstHeight = +firstBar.attr("height");
-        let secondY = +secondBar.attr("y");
-        let secondHeight = +secondBar.attr("height");
-        if (firstY === secondY || Math.abs(secondY + secondHeight - firstY) < 1) {
-          assert.isTrue(firstY === +selection.attr("y1"), "connector is aligned to bars");
-        } else {
-          assert.closeTo(firstY + firstHeight - +selection.attr("y1"), 0, 1, "connector is aligned to bars");
-        }
+      it("can get if connectors are enabled", () => {
+        let connectorsEnabled = true;
```

Comment:
can you merge this test with the "does not have connectors enabled by default"
they seem necessarily small